heft-plugins/heft-browsersync-plugin/src/BrowsersyncPlugin.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ITerminal, Import } from '@rushstack/node-core-library';
+import type {
+  HeftConfiguration,
+  HeftSession,
+  IBuildStageContext,
+  IHeftPlugin,
+  ScopedLogger,
+  IPostBuildSubstage
+} from '@rushstack/heft';
+
+import { configurationFileLoader, IBrowsersyncConfigurationJson } from './BrowsersyncConfigLoader.js';
+
+import type { BrowserSyncInstance } from 'browser-sync';
+
+const bs: typeof import('browser-sync') = Import.lazy('browser-sync', require);
+
+const PLUGIN_NAME: string = 'BrowsersyncPlugin';
+
+interface IBrowsersyncConfigurationFileCacheEntry {
+  configurationFile: IBrowsersyncConfigurationJson | undefined;
+}
+
+/**
+ * @internal
+ */
+export class BrowsersyncPlugin implements IHeftPlugin {
+  public readonly pluginName: string = PLUGIN_NAME;
+
+  private _wdsConfigurationFileCache: Map<string, IBrowsersyncConfigurationFileCacheEntry> = new Map<
+    string,
+    IBrowsersyncConfigurationFileCacheEntry
+  >();
+
+  private _bs?: BrowserSyncInstance;
```

Comment:
```suggestion
  private _browserSyncInstance?: BrowserSyncInstance;
```

Reply:
This looks like it's only used in one function, so it doesn't need to be an instance variable.

Reply:
Agreed, removed in 3660bb4